### ISSUE TITLE
Schematic "interceptor" not found in collection "@ionic/angular-toolkit"

### ISSUE THREAD

**Issue by eduboxgithub** [ionic-team/angular-toolkit#240]
I was trying to add a interceptor using the CLI, but i get this error:

```
ionic g interceptor interceptors/auth
[ERROR] interceptor is not a known feature.
```

If I try to run directly from the Angular CLI:
```
ng generate interceptor interceptors/auth
An unhandled exception occurred: Schematic "interceptor" not found in collection "@ionic/angular-toolkit".
```

Interceptor schematic seems to exist at least in version 9.
**Angular Docs:** https://angular.io/cli/generate#interceptor-command

```
Ionic:

   Ionic CLI                     : 6.6.0 (C:\Users\hsantos\AppData\Roaming\nvm\v13.9.0\node_modules\@ionic\cli)
   Ionic Framework               : @ionic/angular 5.1.1
   @angular-devkit/build-angular : 0.901.6
   @angular-devkit/schematics    : 9.1.6
   @angular/cli                  : 9.1.6
   @ionic/angular-toolkit        : 2.2.0

Capacitor:

   Capacitor CLI   : 2.1.0
   @capacitor/core : 2.1.0

Cordova:

   Cordova CLI       : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms : not available
   Cordova Plugins   : not available

Utility:

   cordova-res : not installed
   native-run  : 0.3.0

System:

   Android SDK Tools : 26.1.1 (C:\Users\hsantos\AppData\Local\Android\Sdk)
   NodeJS            : v13.9.0 (C:\Program Files\nodejs\node.exe)
   npm               : 6.13.7
   OS                : Windows 10
```

**Comment by pmoelgaard** [comment]
no comments on this at all ?

**Comment by itsSoufianeDev** [comment]
I'm facing the same problem, any update?

**Comment by conde2** [comment]
Same problem here, any solution ?

**Comment by Neili-wael** [comment]
same problem here any sol ?

**Comment by VanshilGupta** [comment]
A possible solution worked for me 

Go to _node_modules_ > _@ionic_ > _@angular-toolkit_ > _collection.json_

**Add the interceptor element in the schematics**
![interceptor](https://user-images.githubusercontent.com/63975127/123599293-113b2880-d813-11eb-8b6e-afa7bfc08bfd.PNG)

**Comment by jasonwatkins** [comment]
This is happening to me, but with resolver. I've edited _node_modules > @Ionic > @angular-toolkit > collection.json_ to have the resolver, but no luck.

 I updated local and global angular to latest @angular/core @angular/cli. (12.2.7)

running global `ng g resolver name` in a new angular project on my desktop works fine, but the local version in my ionic project does not have resolver available for some reason.
